modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -577,12 +578,29 @@ struct ReLU6Functor : public BaseFunctor
     }
 #endif  // HAVE_DNN_NGRAPH
 
+
+
 #ifdef HAVE_WEBNN
+    ml::Operand BuildConstant(const ml::GraphBuilder& builder,
+                              const std::vector<int32_t>& dimensions,
+                              const void* value,
+                              size_t size,
+                              ml::OperandType type) {
+        ml::OperandDescriptor desc;
+        desc.type = ml::OperandType::Float32;
+        desc.dimensions = dimensions.data();
+        desc.dimensionsCount = (uint32_t)dimensions.size();
+        return builder.Constant(&desc, value, size);
```

Comment:
Please refer to https://github.com/webmachinelearning/webnn-native/blob/main/examples/SampleUtils.cpp#L62 for the implementation.

Reply:
Hi, Ningxin. I have tried to initialize the `desc` with an initializer-list. However, it failed with the error `c++ could not convert ... from brace-enclosed initializer list to ...`. Thus, I initialized each variable separately instead. I think this is because the version of compiler or c++. And I have fixed the error in Line 590 to fit more types. Thanks!